servers/src/test/java/tachyon/worker/block/TieredBlockStoreTest.java

```diff
@@ -297,4 +298,41 @@ public void freeSpaceWithBlockLockedTest() throws Exception {
     mBlockStore.freeSpace(USER_ID1, mTestDir1.getCapacityBytes(), mTestDir1.toBlockStoreLocation());
     Assert.assertEquals(mTestDir1.getCapacityBytes(), mTestDir1.getAvailableBytes());
   }
+
+  @Test
+  public void getBlockWriterForNonExistingBlockTest() throws Exception {
+    mThrown.expect(NotFoundException.class);
+    mThrown.expectMessage("Failed to get TempBlockMeta: temp blockId " + BLOCK_ID1 + " not found");
+
+    mBlockStore.getBlockWriter(USER_ID1, BLOCK_ID1);
+  }
+
+  @Test
+  public void abortNonExistingBlockTest() throws Exception {
+    mThrown.expect(NotFoundException.class);
+    mThrown.expectMessage("Failed to get TempBlockMeta: temp blockId " + BLOCK_ID1 + " not found");
+
+    mBlockStore.abortBlock(USER_ID1, BLOCK_ID1);
+  }
+
+  @Test
+  public void abortBlockNotOwnedByUserIdTest() throws Exception {
+    mThrown.expect(InvalidStateException.class);
+    mThrown.expectMessage("checkTempBlockOwnedByUser failed: ownerUserId of blockId "
+        + TEMP_BLOCK_ID + " is " + USER_ID1 + " but userId passed in is " + USER_ID2);
+
+    TieredBlockStoreTestUtils.createTempBlock(USER_ID1, TEMP_BLOCK_ID, BLOCK_SIZE, mTestDir1);
+    mBlockStore.abortBlock(USER_ID2, TEMP_BLOCK_ID);
+  }
+
+  @Test
+  public void abortCommitedBlockTest() throws Exception {
+    mThrown.expect(AlreadyExistsException.class);
+    mThrown.expectMessage(
+        "checkTempBlockOwnedByUser failed: blockId " + TEMP_BLOCK_ID + " is committed");
+
+    TieredBlockStoreTestUtils.createTempBlock(USER_ID1, TEMP_BLOCK_ID, BLOCK_SIZE, mTestDir1);
+    mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
+    mBlockStore.abortBlock(USER_ID2, TEMP_BLOCK_ID);
```

Comment:
Should the abort block be by user id 1? Since now we are relying on the order of errors to matter?